### ISSUE TITLE
heatmap: Option to transpose the plot

### ISSUE THREAD

**Issue by etal** [etal/cnvkit#35]
The current "horizontal" layout shows samples in rows, chromosomes in columns. A transposed or "vertical" layout would show samples in columns, chromosomes in rows. (Sometimes this makes a better figure.)

Consider the flags "-t/--transpose" or "-v/--vertical".

**Comment by tetedange13** [comment]
Hi @etal ,
<br>

I do not know if this is still a desired new feature, but I came up with something quite satifying: https://github.com/tetedange13/cnvkit/tree/35-vertical-heatmap
1. I first tried keeping up with `BrokenBarHCollection`, but verticalized plot was quite heavy to plot (loading all the time, zooming locked...)
2. Then I implemented a 2nd version using `pcolor` + a native Matplotlib colormap

=> Not perfect, still some polish probably
=> Also I could not reimplement `--desaturate` mode as I had trouble understanding the maths...
=> IMHO it should be possible to implement it directly inside the colormap using [custom normalization](https://matplotlib.org/stable/tutorials/colors/colormapnorms.html#funcnorm-arbitrary-function-normalization) (with a proper sigmoid function)
<br>

Hope this helps.
Have a nice day.
Felix.
